code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/react-in-jsx-scope */
+import './reset.css';
+import './styles.css';
+import './dark-theme.css';
+
+import { useState, useEffect } from 'react';
+
+import Message from './components/Message';
+import Submission from './components/Submission';
+
+export default function App() {
+  const [messages, setMessages] = useState([]);
+  const [areLiked, setAreLiked] = useState([]);
+  const [needsUpdating, setNeedsUpdating] = useState(false);
+
+  function themeSwitcher() {
+    if (
+      window.matchMedia
+      && window.matchMedia('(prefers-color-scheme: light)').matches
+    ) {
+      document.body.classList.toggle('is-forced-dark');
+    } else if (
+      window.matchMedia
+      && window.matchMedia('(prefers-color-scheme: dark)').matches
+    ) {
+      document.body.classList.toggle('is-forced-light');
+    }
+  }
+
+  useEffect(() => {
+    // Ropa på api asynkront
```

Comment:
We should avoid non-english comments. Let's say that english language is developer's latin, we should communicate and inform others only in this language